guides/rails_guides/markdown/renderer.rb

```diff
@@ -89,7 +89,7 @@ def lexer_language(code_type)
           when "html+erb"
             "erb"
           when "bash"
-            "console"
+            "console?comments=true"
```

Comment:
It seems that the method is also used for adding html classs https://github.com/rails/rails/pull/54316/files#diff-f87bbf21d7ba3fd29686f904036a4f1aab7b4a435dd93045b05b19428bc89df7R39
and it will now generate a `<code class="highlight console?comments=true">`, is this ok?

Reply:
Ah, nice catch. I'll change it.